journals/settings/production.py

```diff
@@ -26,5 +26,9 @@
     PORT=environ.get('DB_MIGRATION_PORT', DATABASES['default']['PORT']),
 )
 
+WAGTAILSEARCH_BACKENDS['default'].update({
+    'URLS': ['http://localhost:9200']
```

Comment:
This should pull an environment variable similar to the other settings in prod. Because when we release it to actual product (not sandbox) this URL won't be correct.